Building from .zip should be possible

Currently you cannot build Prometheus using the zip-files mentioned in fe. https://github.com/prometheus/prometheus/releases/tag/0.12.0.

When you try you get a bunch of warnings such as `fatal: Not a git repository (or any parent up to mount point /tmp)` eventually followed by the usage output of 6l and the failure `Makefile:30: recipe for target 'build' failed`.

Relevant chat on IRC:
> jrv | ah, the zip file doesn't work because the Makefile expects the source directory to still be a git repository.
> jrv | GitHub creates those zip files automatically, so we never tried that.
> jrv | "-X main.buildRevision" isn't followed by an argument for that reason, which is why ld6 crashes
> jrv | cause that's built via a git command
> jrv | VERSION=$(shell cat `git rev-parse --show-toplevel`/VERSION)

--- a/Makefile.INCLUDE
+++ b/Makefile.INCLUDE
@@ -15,7 +15,7 @@
 
 .SUFFIXES:
 
-VERSION=$(shell cat `git rev-parse --show-toplevel`/VERSION)
+VERSION?=$(shell cat `git rev-parse --show-toplevel`/VERSION)
 
 OS=$(shell uname)
 ARCH=$(shell uname -m)

